src/js/quiz-component.js

```diff
@@ -0,0 +1,75 @@
+const quizComponent = (quizInfo) => {
+    const title = document.createElement("h2");
+    title.id = "title";
+    title.textContent = "Pop quiz!";
+  
+    const question = document.createElement("p");
+    question.id = "question";
+    question.textContent = quizInfo.question;
+
+    const submitBtn = document.createElement("button");
+    submitBtn.id = "submit-btn";
+    submitBtn.textContent = "Submit my answer!";
+    
+    // Here goes the function declared in the game logic module. Which handles the good and bad answers.
+    // submitBtn.addEventListener("click", answerHandler);
+
+    const quiz = document.createElement("div");
+    quiz.id = "quiz";
+    quiz.appendChild(title);
+  
+    const form = document.createElement("form");
+    form.id = "answers";
+
+    const nextChapterBtn = document.createElement("button");
+    nextChapterBtn.id = "next-chapter-btn";
+    nextChapterBtn.textContent = "Next chapter!";
+
+    // Here goes the function that links to the next chapter.
+    // nextChapterBtn.addEventListener("click", nextChapterFunction);
+
+    /* Passed is defined in the gamelogic component.
+    Will need to change the way it's called here. */
+    if (passed) {
+        nextChapterBtn.style.display = "block";
```

Comment:
If possible do styling stuff in css --> You can accomplish it here with creating a class in our css file: 
```
.hide {
  display: none;
}
```

And in quiz-component you work with it like this:
```
nextChapterBtn.classList.add('hide')
nextChapterBtn.classList.remove('hide')
```
Or what I discovered lately: 
`nextChapterBtn.classList.toggle('hide')`